client/reader/post-time/index.jsx

```diff
@@ -3,48 +3,17 @@
  * External dependencies
  */
 import React from 'react';
-import { PureComponent } from 'react';
-import moment from 'moment';
+import { PropTypes } from 'prop-types';
+
 /**
  * Internal dependencies
  */
-import smartSetState from 'lib/react-smart-set-state';
-import ticker from 'lib/ticker';
-import humanDate from 'lib/human-date';
-
-export default class PostTime extends PureComponent {
-	smartSetState = smartSetState;
-
-	componentWillMount() {
-		this.update();
-	}
-
-	componentDidMount() {
-		ticker.on( 'tick', this.update );
-	}
-
-	componentWillReceiveProps( nextProps ) {
-		this.update( nextProps.date );
-	}
+import TimeSince from 'components/time-since';
```

Comment:
why not have the whole file be: 
```js
export default TimeSince from 'components/time-since';
```

Reply:
Good shout! - that could be a good intermediate step before the follow up PR that handles importing `time-since` directly throughout the files.

Reply:
changing filepaths in js apps is terrifying.  I definitely think it should be split into two steps